pyro/contrib/gp/models/gpr.py

```diff
@@ -62,26 +75,18 @@ def guide(self):
         return kernel, noise
 
     def forward(self, Xnew, full_cov=False, noiseless=True):
-        r"""
-        Computes the parameters of :math:`p(y^*|Xnew) \sim N(\text{loc}, \text{cov})`
-        w.r.t. the new input :math:`Xnew`. In case output data is a 2D tensor of shape
-        :math:`N \times D`, :math:`loc` is also a 2D tensor of shape :math:`N \times D`.
-        Covariance matrix :math:`cov` is always a 2D tensor of shape :math:`N \times N`.
-
-        :param torch.Tensor Xnew: A 1D or 2D tensor.
-        :param bool full_cov: Predicts full covariance matrix or just its diagonal.
-        :param bool noiseless: Includes noise in the prediction or not.
-        :returns: loc and covariance matrix of :math:`p(y^*|Xnew)`.
-        :rtype: torch.Tensor and torch.Tensor
+        """
+        :param bool noiseless: A flag to decide if we want to include noise in the
+            prediction output or not.
```

Comment:
ditto: Why did you delete this docstring?

Reply:
I found that it is redundant to repeat docs in derived classes. I will add them again if you think that it is a good idea.

I also reduced docs for classes in `gp.kernels`, do you think that we should not reduce there too?